components/media/VideoThumbnail.tsx

```diff
@@ -35,7 +35,8 @@ const VideoThumbnail = ({ thumbnailUrl, onPress, width, height }: Props) => {
           <IconButton
             icon={
               <Icon
-                as={<Ionicons name="play-circle" />}
+                as={Ionicons}
+                name="play-circle"
```

Comment:
note that this is purposefully omitted from the migration, as the icon is always on top of media and should stay white/transparent.